### ISSUE TITLE
aria-relevant link on ARIA roles landing page broken

### ISSUE THREAD

**Issue by andywolff** [mdn/sprints#1187]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

Clicking on the aria-relevant link in [the aria roles landing page]( https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques) brings you to a ["not found" page]( https://developer.mozilla.org/en-US/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-relevant_attribute ) instead of the [appropriate page]( https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-relevant_attribute ). It's missing the docs/Web/ part of the URL which other links on the page seem to add automatically somehow.

**Comment by chrisdavidmills** [comment]
Thanks for reporting. I've fixed it now.
